Removed unwanted NoWarns.

Timovzl opened the pull request.

Timovzl committed:
Removed unwanted NoWarns.

TheArchitectDev reviewed (changes requested):
Increment Version please.

Timovzl committed:
Incremented version and added (empty) ReleaseNotes.

Updated readme and removed NoWarns. These themselves did not seem ReleaseNotes-worthy.

Timovzl committed:
Incremented Version.

Forgot last time.
